dashboard/app/controllers/projects_controller.rb

```diff
@@ -589,9 +589,12 @@ def limited_gallery?
   # Redirect to the correct view/edit page for Lab2 projects. If a project owner is on a /view
   # page, redirect to /edit. If a non-owner is on an /edit page, redirect to /view.
   # For legacy (non-Lab2) labs, this is handled on the front-end.
-  private def redirect_edit_view_for_lab2
+  private def redirect_view_for_lab2
     return nil unless @level.uses_lab2?
-
+    # If the user is on the play view '/projects/channel_id', set `response_content`.`
+    if params[:share] == true
```

Comment:
Not sure if this is okay to do this for `lab2` labs or should I only do for music lab projects for now? @sanchitmalhotra126